### ISSUE TITLE
Drag and Drop crashes

### ISSUE THREAD

**Issue by RELnemtzov** [projectstorm/react-diagrams#683]
I have observed this issue when trying to integrate `react-diagrams` within my app initialized with `react-boilerplate`.

I have been able to reproduce the issue in the following CodeSandbox: https://codesandbox.io/s/react-diagrams-drag-and-drop-crash-el9d2

This doesn't seem to happen in an app initialized with create-react-app, as can be seen in the following CodeSandbox: https://codesandbox.io/s/react-diagrams-drag-and-drop-success-ti7cw

Since these both contain the same code aside for the `webpack` and `babel` configurations this leads me to believe that it is not something specific to `react-boilerplate`.

The specific error messages I received in the console are 
```Uncaught TypeError: a.default is not a constructor at l.componentDidMount (index.js?d9df:9)...```
and
```Uncaught TypeError: Cannot read property 'disconnect' of undefined at l.componentWillUnmount (index.js?d9df:9)...```

This seems to be related to https://github.com/projectstorm/react-diagrams/issues/599 hence my comment there, but seems to be broader then described there, so I opened a new issue.

**Comment by renato-bohler** [comment]
Seems like the same problem faced in #680

**Comment by RELnemtzov** [comment]
@renato-bohler seems like the same error, but this doesn't occur in tests, so I can't solve it by mocking the `resize-observer-polyfill` library as @zx-zheng did there.

**Comment by renato-bohler** [comment]
> @renato-bohler seems like the same error, but this doesn't occur in tests, so I can't solve it by mocking the `resize-observer-polyfill` library as @zx-zheng did there.

Yeah, sorry for the confusion: by the time I replied here, he still hadn't figured out his issue... I was just linking both issues :sweat_smile:
